smarc_keyboard_teleop/scripts/lolo_auv_teleop_joystick.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/python
```

Comment:
The above binary file should not be added to the repo.

Reply:
I.e. `smarc_keyboard_teleop/scripts/.lolo_auv_teleop.py.swp`

Reply:
I don't even know where that came from